function consoleError1(errmsg) {
  console.log(errmsg);
}
function handleError1(errmsg) {
  throw new Error(errmsg);
}
